src/.swcrc

```diff
@@ -1,5 +1,6 @@
 {
     "jsc": {
+        "target": "es2019",
```

Comment:
Doesn't this change break old node versions?

Reply:
CLI declare node > 12.13 It should be safe to target ES2019 https://stackoverflow.com/questions/59787574/typescript-tsconfig-settings-for-node-js-12

This package is published with swc or with typescript compiled output?

Reply:
Currently, it uses typescript.

Reply:
tsc already target es2019.